spec/problem3_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'problem3'
+describe 'problem3' do
+
+  it 'returns true on arrays with duplicates' do
+    expect(problem3([1,2,2])).to eq true
+    expect(problem3([5,4,4,2,1])).to eq true
```

Comment:
In unit tests, it's preferable to use a single expectation per test http://www.betterspecs.org/. Can also make use Rspec's shared examples

Reply:
I'll take time to read the shared examples section, because from a quick glance I'm not yet sure how they would be applied in this case